dashboard/app/helpers/script_levels_helper.rb

```diff
@@ -36,9 +38,21 @@ def next_progression_level_or_redirect_path(script_level)
         script_level.next_progression_level
       end
 
-    next_level ?
-        build_script_level_path(next_level) :
-        script_completion_redirect(script_level.script)
+    if script_level.level.try(:is_plc_evaluation?)
+      last_result = UserLevel.where(user: current_user, level: script_level.levels[0]).last
```

Comment:
So it's possible for there to be multiple user levels, per user and level. But if I pass in the script then this is unique